src/components/containers/BooksList.js

```diff
@@ -1,47 +1,85 @@
-/* eslint-disable react/forbid-prop-types */
```

Comment:
- [ ] We can safely remove this eslint disable line. Let's remove it before merging the PR